index.html

```diff
@@ -338,9 +339,47 @@ <h5 class="language">Skills</h5>
                               </div>
                         </div>
                     </div> 
-                </section>
-          </footer> 
+     
+                <!-- </section> -->
+                <!-- ---------------------------------------------------------------End----------------------------------------------------------------- -->
+                
+      </section> 
+           <!-- -------------------------------------------------------------contact form ---------------------------------------------------------- -->
+           <section class="contact-section ">
+            <div class="contact-content">
+                    <div class="contact-me">
+                      <h2>contact me</h2>
+                    </div>
+                    <div class="about-us">
+                          <p>
+                            If you have an application you are interested in developing, 
+                            a feature that you need built or a project that needs coding. 
+                            I’d love to help with it!
+                          </p>
+                    </div>
+              
+                  <form action="https://formspree.io/f/{form_id}" method="post">
+                      <div>
+                        <input type="text" name="userName" maxlength="30" placeholder="userName" required>
+                      </div>
+                      <div>
+                        <input type="mail" name="email" placeholder="example@gmail.com" required>
```

Comment:
- [ ] Ouch! this is so close the `type` attribute for the email input should be `email` instead of `mail` so kindly make sure you fix that. This will make sure the user follows the proper email format. 👮‍♂️